lib/spack/env/cc

```diff
@@ -98,25 +98,25 @@ case "$command" in
     cpp)
         mode=cpp
         ;;
-    cc|c89|c99|gcc|clang|icc|pgcc|xlc)
+    cc|c89|c99|gcc|clang|icc|pgcc|xlc|xlc_r)
```

Comment:
This is how I add the fflags in my xl_r class
    @property
    def fflags(self):
        return "-qzerosize -qfixed"

They don't show up under flags in compilers.yaml.  Is it the right way to specify them?